src/lib/Asset.ts

```diff
@@ -58,9 +58,8 @@ class Asset {
   }
 
   /**
-   * Get expiring multi party (EMP) state
-   *
-   * @return A promise with the info of the metapool contract
+   * @notice Get expiring multi party (EMP) state.
```

Comment:
Oh nice. Thanks!

Reply:
Preparing for the changes that I made in #86.